### PR TITLE
Kernel migration to 5.2.0-alpha

### PR DESCRIPTION
**Pull request by grainier**

### CODE REVIEW

File: modules/distribution/carbon-home/conf/editor/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,205 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (this) [inline modules/distribution/carbon-home/conf/editor/deployment.yaml]: Year should be 2017 in the license header

File: modules/distribution/carbon-home/conf/manager/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,205 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (this) [inline modules/distribution/carbon-home/conf/manager/deployment.yaml]: Year should be 2017 in the license header

File: modules/distribution/carbon-home/conf/worker/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,205 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (this) [inline modules/distribution/carbon-home/conf/worker/deployment.yaml]: Year should be 2017 in the license header

File: modules/distribution/carbon-home/conf/editor/deployment.yaml (hunk 1 above)
Comment (grainier) [reply modules/distribution/carbon-home/conf/editor/deployment.yaml]: fixed

File: modules/distribution/carbon-home/conf/manager/deployment.yaml (hunk 2 above)
Comment (grainier) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: fixed

File: modules/distribution/carbon-home/conf/worker/deployment.yaml (hunk 3 above)
Comment (grainier) [reply modules/distribution/carbon-home/conf/worker/deployment.yaml]: fixed

File: modules/distribution/carbon-home/bin/dashboard.bat [hunk 4]
```diff
@@ -18,7 +18,7 @@ REM   limitations under the License.
 rem ---------------------------------------------------------------------------
 rem Main Script for WSO2 Carbon
 rem
-rem Environment Variable Prequisites
+rem Environment Variable Perquisites
```
Comment (gokul) [inline modules/distribution/carbon-home/bin/dashboard.bat]: Prerequisites

**Pull request merged**